src/lib/code-sample-editor-preview.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {
+  customElement,
+  LitElement,
+  html,
+  css,
+  property,
+  query,
+} from 'lit-element';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import '@material/mwc-icon-button';
+import '@material/mwc-textfield';
+
+@customElement('code-sample-editor-preview')
+export class CodeSampleEditorPreviewElement extends LitElement {
```

Comment:
I should add a doc comment. This is a component responsible for setting up the iframe, and has a URL bar-like header with the file name and a reload button. It's intended to evolve to a more full-featured preview component, maybe with a console log viewer, drop-down to select different html files to view, etc.